### PR TITLE
Naming "compose()" an optional function.

### PR DESCRIPTION
**Pull request by koresar**
See #14 for more details.

### CODE REVIEW

**Comment by unstoppablecarl** [comment]
looks good to me

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Composables specification
 
-## Introduction
+## Compatibility between different implementations
```
Comment (ericelliott) [inline README.md]: I don't think this should be at the heading level. Instead, try:

## Introduction

The composables specification exists in order to define a standard format for composable factory functions (called **stamps**). It exists to ensure compatability between different stamp implementations.

Comment (troutowicz) [inline README.md]: :+1:

**Comment by troutowicz** [comment]
Other than @ericelliott's suggestion, this looks good.

**Comment by koresar** [comment]
Thank you guys for the input.
Sorry It took me ages to ge tback to this issue.

**Pull request merged**